### PR TITLE
Adds GPU related variables to VSphere Cluster Class + CAPV bump v1.5.3

### PR DESCRIPTION
**Pull request by srm09**
### What this PR does / why we need it
Upgrades CAPV version to v1.5.3 to incorporate fixes

### Which issue(s) this PR fixes

Fixes #

### Release note

```release-note

```

### CODE REVIEW

File: tkg/client/upgrade_region.go [hunk 1]
```diff
@@ -260,6 +260,20 @@ func (c *TkgClient) validateAndconfigure(options *UpgradeClusterOptions, regiona
...
+		if err != nil {
+			return NewValidationError(ValidationErrorCode, errors.Wrap(err, "failed to get VC client").Error())
+		}
+
+		dc, err := c.TKGConfigReaderWriter().Get(constants.ConfigVariableVsphereDatacenter)
+		if err != nil {
+			return NewValidationError(ValidationErrorCode, errors.Errorf("failed to get %s", constants.ConfigVariableVsphereDatacenter).Error())
+		}
+
+		if err := c.ValidateVsphereResources(vcClient, dc); err != nil {
```
Comment (DanielXiao) [inline tkg/client/upgrade_region.go:274]: We need to figure out when and how the full path values are changed in the secret tkg-pkg-tkg-system-values. The code logical is suppose to reuse the same value with full path.

File: addons/go.mod [hunk 2]
```diff
@@ -36,7 +36,7 @@ require (
 	k8s.io/utils v0.0.0-20220812165043-ad590609e2e5
 	knative.dev/pkg v0.0.0-20220302134643-d2cdc682d974
 	sigs.k8s.io/cluster-api v1.2.8
-	sigs.k8s.io/cluster-api-provider-vsphere v1.5.0
+	sigs.k8s.io/cluster-api-provider-vsphere v1.5.2
```
Comment (wjun) [inline addons/go.mod:39]: Why is it 1.5.2 here?

Comment (srm09) [reply addons/go.mod]: Addressed, thanks.
Previously this same PR was bumping CAPV to 1.5.2, updated it to point to v1.5.3. Missed one of the imports, thanks!

**Review by wjun — APPROVED** [review]
LGTM

**Comment by alfredthenarwhal** [comment]
Created cherry-pick PR

**Pull request merged**